docs/validators-api.md

```diff
@@ -495,3 +495,27 @@ Arguments:
 | "pepsi"               | ["coca-cola", "pepsi"] | True      |
 | "rc-cola              | ["coca-cola", "pepsi"] | False     |
 
+## isPercentage
+
+isPercentage validates that a given string is a decimal OR whole number between 0 and 100. If a decimal is included, only tenths and hundredths place values are allowed.
```

Comment:
why only 2 decimal places? feels weird because 1.238 _is_ a value percentage. I don't wanna make more work for you, but it seems like the amount of decimal places would be a separate validator?

Reply:
or it could be an option for this validator to limit decimal places? (not sure if that's possible)

Reply:
that's a fair point - i'm enforcing something here just to match cb pos that should be configurable. there are a lot of permutations of this and it's difficult to find an appropriate name. like technically percentages can also be negative or over 100%. something like `fractionOf100` would be more accurate but I don't love that name either. 

i'm hoping this can serve as a starting point that could be iterated on in the future if that's needed by another application. i can see why another validator might be preferred but passing a param seems like a much lighter weight solution for this if that works for you?

Reply:
yeah this is difficult to name. are there already validators you could use for min/max values? then you just need one to enforce decimal places

Reply:
that approach does sound more broadly applicable and will narrow this in scope. i should be able to do `numberGreaterThanOrEqualTo(0), numberLessThanOrEqualTo(100)` plus the new one